packages/design-system/src/components/Review/Review.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import ReviewLink from './ReviewLink';
+import classNames from 'classnames';
+
+export type ReviewHeadingLevel = '1' | '2' | '3' | '4' | '5';
+
+export interface ReviewProps {
+  /**
+   * `Review` component's body content.
+   */
+  children: React.ReactNode;
+  className?: string;
+  /**
+   * Optional label to give screenreaders longer, more descriptive text to
+   * explain the context of an edit link.
+   */
+  editAriaLabel?: string;
+  /**
+   * An optional node in place of the edit link. If this defined, no edit link will be shown.
+   */
+  editContent?: React.ReactNode;
+  /**
+   * Href for the edit link. If this is undefined, no edit link will be shown.
+   */
+  editHref?: string;
+  editText?: React.ReactNode;
+  heading?: React.ReactNode;
+  /**
+   * Heading type to override default `<h3>`.
+   */
+  headingLevel?: ReviewHeadingLevel;
+  /**
+   * An optional function that is executed on edit link click. The event and
+   * props.editHref value are passed to this function.
+   */
+  onEditClick?: (...args: any[]) => any;
+}
+
+export const Review = (props: ReviewProps) => {
+  const getHeading = () => {
+    const Heading = (`h${props.headingLevel}` || `h3`) as keyof JSX.IntrinsicElements;
+    if (props.heading) {
+      return <Heading className="ds-c-review__heading">{props.heading}</Heading>;
+    }
+  };
```

Comment:
Up to you, but this could be moved out of the component with a parameter now that you've made it into a nice little function.